website/pages/en/index.js

```diff
@@ -77,7 +77,7 @@ class HomeSplash extends React.Component {
     const language = this.props.language || '';
     return (
       <SplashContainer>
-        <Logo img_src={imgUrl('docusaurus.svg')} />
+        {/* <Logo img_src={imgUrl('docusaurus.svg')} /> */}
```

Comment:
Because as long as I"m working with Docusaurus I don't want to delete everything then have to relearn from scratch how to do simple things.  When we get a design maybe I can delete this.  But, for now, my preference is the keep this as a comment.

Reply:
Can we add a comment to that effect to notify others who may want to contribute documentation?